### PR TITLE
updated splunk add-on README.md

### PR DESCRIPTION
**Pull request by dantavori**
relates: https://github.com/demisto/etc/issues/29616

### CODE REVIEW

**Review by roysagi — APPROVED** [review]
Looks good to me but since this is highly public I think we should let Alex check this - @glicht what do you think?

File: add-on/README.md [hunk 1]
```diff
@@ -1,56 +1,135 @@
-
-# ABOUT THIS APP
-
+# Demisto Add-on for Splunk
 Supporting Add-on for Cortex XSOAR. This application allows a user to create incident into XSOAR from Splunk using custom alert action.
```
Comment (glicht) [inline add-on/README.md]:
```suggestion
Supporting Add-on for Cortex XSOAR. This application allows a user to push incidents into Cortex XSOAR from Splunk using a custom alert action.
```

File: add-on/README.md [hunk 2]
```diff
@@ -1,56 +1,135 @@
...
-
-* Splunk version 6.3 >=
-* This application should be installed on Search Head.
-
+# Requirements
+* Splunk version 7.2 >=
 
-# Recommended System configuration
-* Standard Splunk configuration of Search Head.
+# Procedure for Using the Demisto Add-on in Splunk
```
Comment (glicht) [inline add-on/README.md]:
```suggestion
# Development Setup for Using the Demisto Add-on in Splunk
```

**Review by glicht — APPROVED** [review]
Looks good. See my small comments

**Comment by glicht** [comment]
> Looks good to me but since this is highly public I think we should let Alex check this - @glicht what do you think?



> Looks good to me but since this is highly public I think we should let Alex check this - @glicht what do you think?

It is ok as is. It is for developers so I am ok with merging.

**Pull request merged**